authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -4,6 +4,7 @@
 from rest_framework.test import APITestCase, APIClient
 from unittest import skip
 
+
```

Comment:
@RachelleMaina, does this change contribute to the PT board story?

Reply:
It does.

Reply:
@abulojoshua1 I misunderstood the question, sorry about that. No, they do not. These changes were rather unnecessary. They came as a result of merge conflicts after rebasing. I may not have been keen enough. I will avoid such changes in the next PRs